contracts/Colony.sol

```diff
@@ -124,7 +124,16 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
   stoppable
   auth
   {
-    return token.mint(_wad);
+    token.mint(_wad);
+  }
+
+  // TODO: remove this in favor of the token authority (see colonyNetwork#387).
+  function mintTokensFor(address _dst, uint _wad) public
```

Comment:
Why leave the TODO here? We shouldn't need this function if you use the Authority contract

Reply:
Yup! Going to remove it imminently.

Reply:
The whole function right?

Reply:
Like it was never there